Brokerages/Backtesting/BacktestingBrokerage.cs

```diff
@@ -457,5 +480,27 @@ private void SetPendingOrder(Order order)
             // only save off clones!
             _pending[order.Id] = order.Clone();
         }
+
+        private ITimeInForceHandler GetTimeInForceHandler(TimeInForce timeInForce)
```

Comment:
instead of having this function, it might be more concise to simply initialize the dictionaries with all the handlers already mapped, and then replace calls to this function with explicit dictionary access, `_timeInForceHandlers[timeInForce]`